.github/workflows/test-tutor-aspects.yml

```diff
@@ -37,8 +37,12 @@ jobs:
         run: tutor images build openedx aspects aspects-superset
       - name: Tutor start
         run: tutor local start -d
+      - name: Test commands
```

Comment:
Maybe we can do a downgrade to base, the upgrade to head? That will test all of the migrations too. Or at least that they don't throw errors.